src/ui/render_config.rs

```diff
@@ -193,6 +200,7 @@ impl RenderConfig {
             unselected_checkbox: Styled::new("[ ]"),
             option_index_prefix: IndexPrefix::None,
             option: StyleSheet::empty(),
+            selected_option: StyleSheet::empty(),
```

Comment:
Wdyt of making the default green colored?

I think it makes more sense, it is easier to see what the current option is and everything.

I'm completely fine in releasing a new minor version to signal a "breaking contract".

Reply:
~how opinions change in 2 weeks~